dm/tests/dmctl_command/conf/dm-task.yaml

```diff
@@ -54,7 +54,7 @@ validators:
   global:
     mode: full
     worker-count: 1
-    validate-interval: 1s
-    check-interval: 1s
-    row-error-delay: 3s
-    meta-flush-interval: 3s
+    validate-interval: 1ms
+    check-interval: 1ms
+    row-error-delay: 1ns
+    meta-flush-interval: 1ms
```

Comment:
prefer not to change it too small, maybe stick to 1s

for unstable case, add sleep before last ddl used to trigger flush is enough in most case, see i t in validator_basic/validator_checkpoint